### PR TITLE
Doc adjustments for v1

### PR DESCRIPTION
**Pull request by jwpereira**
With us heading into the marketplace real soon, I want to make sure the README will reflect the v1 status and latest discussions on naming 🤔

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ jobs:
...
       - name: Run script
-        uses: matlab-actions/run-command@v0
+        uses: matlab-actions/run-command@v1
         with:
           command: myscript
 ```
 
 ### Run MATLAB Commands on GitHub-Hosted Runner
-Before you run MATLAB code or Simulink models on a GitHub-hosted runner, first use the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action installs your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB.
+Before you run MATLAB code or Simulink models on a GitHub-hosted runner, first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action installs your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB.
```
Comment (acampbel) [inline README.md:32]: The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.

**Review by acampbel — APPROVED** [review]
:shipit:

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ The [Run MATLAB Command](#run-matlab-command) GitHub&reg; action enables you to
 
 - To use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the topmost MATLAB version on the runner's system path.
 
-- To use a GitHub-hosted runner, you must include the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to install MATLAB on the runner. Currently, this action is available only for public projects. It does not install transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to install MATLAB on the runner. Currently, this action is available only for public projects. It does not install transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
```
Comment (mw-hrastega) [inline README.md:7]: How about the "install" instances in this paragraph? 🤒

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md:32]: Good comment. Let's replace all "install" instances.

File: README.md (hunk 2 above)
Comment (acampbel) [reply README.md:7]: > How about the "install" instances in this paragraph? 🤒

good catch! lets say "does not include..."

**Review by mcafaro — APPROVED** [review]
+1 replace all instances of "install" with "set up"

**Review by mw-hrastega — APPROVED** [review]

**Comment by jwpereira** [comment]
Oops 😂 
I thought I had all three repos lined up with the same changes 🤯

**Pull request merged**